src/storage/storage.h

```diff
@@ -48,7 +48,6 @@
 
 enum class StorageEngineType : uint16_t {
   RocksDB,
-  Speedb,
```

Comment:
So we don't need a `1` for legacy speedb here?

Reply:
Yes, I guess no. We added this to allow awareness of the storage type so that we can tune the performance for different storage types, but we haven't used it for now.